FLEDGE.md

```diff
@@ -255,6 +274,8 @@ else
 
 This will cause the browser to execute the appropriate bidding and auction logic inside a collection of dedicated worklets associated with the buyer and seller domains.  The `auctionSignals`, `sellerSignals`, and `perBuyerSignals` values will be passed as arguments to the appropriate functions that run inside those worklets — the `auctionSignals` are made available to everyone, while the other signals are given only to one party.
 
+The optional `requestedSize` field recommends a frame size for the auction, which will be available to bidders in browser signals. Bidders inside the auction may pick a different size, but that resulting size will be scaled to fit inside the requested size. The returned fenced frame config will automatically populate a `<fencedframe>` with the right size when loaded, unless the size is overridden with the element attributes.
```

Comment:
> unless the size is overridden with the element attributes

In this case is the ad scaled from the `requestedSize` to the element's size?

Reply:
`requestedSize` is just a suggestion from the auction; it sets the `width` and `height` attributes on the `<fencedframe>`. But you can set the attributes to other values afterward. It's the `renderSize` that determines the FF content's frozen size, and the `renderSize`-size content will be scaled visually to fit in whatever size the `<fencedframe>` attributes have.

Reply:
Updated description to clarify this